test/Interop/Cxx/class/extensions-silgen.swift

```diff
@@ -0,0 +1,9 @@
+// RUN: %target-swiftxx-frontend -I %S/Inputs -emit-silgen %s | %FileCheck %s
+
+import Extensions
+
+extension Outer.Space.Foo {
+  func foo() {}
+}
+
+// CHECK: sil hidden [ossa] @$sSo5OuterO5SpaceO10ExtensionsE3FooV4mainE3fooyyF : $@convention(method) (Outer.Space.Foo) -> () {
```

Comment:
Do you think it would make sense to make this an IRGen test instead? And maybe check that we properly call the function too?

Reply:
Yeah, we can have an IRGen test instead. Added a call as well.